File: pset6/mario/less/mario.py
altura = 0
while altura < 1 or altura > 8:
    try:
        altura = int(input("Tamanho: "))
    except:
        print("Informe um número entre 1 e 8")

for i in range(altura):
    for j in range(altura):
        print(" " if j < (altura - (i + 1)) else "#", end="")
    print()

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int altura;
    do
    {
        altura = get_int("Tamanho: ");
    }
    while (altura < 1 || altura > 8);

    for (int i = 0; i < altura; i++)
    {
        for (int j = 0; j < altura; j++)
        {
            printf((j < (altura - (i + 1))) ? " " : "#");
        }

        printf("  ");

        for (int m = i; m >= 0; m--)
        {
            printf("#");
        }
        printf("\n");
    }
}